dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/jsonpatch/BulkPatchParameters.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2004-2004-2021, University of Oslo
```

Comment:
Should be `2004-2021` :) several places

Reply:
fixed. there are some more files have same issue, i will update other files in another PR